src/pages/UploadPage.tsx

```diff
@@ -70,16 +123,17 @@ export default function UploadPage() {
           component="label"
           sx={{ width: '100px' }}
           onClick={sendFiles}
+          disabled={isSendButtonDisabled}  // Disable the button conditionally
```

Comment:
```suggestion
          disabled={isSendButtonDisabled}  // Disable the button conditionally
          disableElevation
```
Disable elevation to keep a consistent button look throughout the application.